packages/metal-dom/src/domNamed.js

```diff
@@ -340,7 +340,18 @@ export function hasClass(element, className) {
  * @private
  */
 function hasClassWithNative_(element, className) {
-	return element.classList.contains(className);
+	try {
```

Comment:
try/catch won't be optimized in browsers, would be better to check for presence of `element.classList`.